test/skimNtuple2018.cpp

```diff
@@ -2344,6 +2430,44 @@ int main (int argc, char** argv)
           idAndIsoSF_leg2_deep_vsEle    = Deep_antiEle_vloose   ->getSFvsEta(tau2eta, tau2Genmatch);
           idAndIsoSF_leg2_deep_vsMu     = Deep_antiMu_tight     ->getSFvsEta(tau2eta, tau2Genmatch);
 
+          vector <float> idAndIsoSF_leg2_deep_vsJet_up     (idAndIsoSF_leg2_deep_vsJet, 3); // in bins of DM: 0, 1, 10/11 
+          vector <float> idAndIsoSF_leg2_deep_vsJet_pt_up  (idAndIsoSF_leg2_deep_vsJet, 5); // in bins of pt: 20, 25, 30, 35, 40, infty
+          vector <float> idAndIsoSF_leg2_deep_vsEle_up     (idAndIsoSF_leg2_deep_vsEle, 2); // in bins of eta: barrel, endcap
+          vector <float> idAndIsoSF_leg2_deep_vsMu_up      (idAndIsoSF_leg2_deep_vsMu,  4); // in bins of eta, edges at 0, 0.4, 0.8, 1.2, 1.7, infty
+
+          vector <float> idAndIsoSF_leg2_deep_vsJet_down     (idAndIsoSF_leg2_deep_vsJet, 3); // in bins of DM: 0, 1, 10/11 
+          vector <float> idAndIsoSF_leg2_deep_vsJet_pt_down  (idAndIsoSF_leg2_deep_vsJet, 5); // in bins of pt: 20, 25, 30, 35, 40, infty
```

Comment:
here should be `idAndIsoSF_leg2_deep_vsJet_pt` instead of `idAndIsoSF_leg2_deep_vsJet`